ddp-workspace/projects/ddp-dsm-ui/src/app/FON/layout/navigation/navigation.component.ts

```diff
@@ -28,4 +30,9 @@ export class NavigationComponent {
     const authToken = localStorage.getItem(Auth.AUTH0_TOKEN_NAME);
     return this.sessionService.getDSMClaims(authToken).name || 'Administrator name';
   }
+
+  isActive(route: string): boolean {
+    const [first,,third] = this.activatedRoute.snapshot['_routerState']['url'].split('/');
+    return third ? route === third : route === first;
```

Comment:
**_#minor_**

it would be nice to have more readable names here instead of first/third

Reply:
This TS file I will change later completely. It was just for activating class according to route (that still doesn't work properly). I might include changes for that in this PR. I Will see about the time.